pyfar/classes/audio.py

```diff
@@ -637,27 +624,27 @@ def time(self):
 
     @FrequencyData.freq.getter
     def freq(self):
-        """Return the data in the frequency domain."""
-        # converts the data from 'time' to 'freq'
-        self.domain = 'freq'
-
-        return super().freq
+        """Return the (normalized) frequency domain data."""
```

Comment:
I would suggest to remove the parentheses around 'normalized' and to add a second line to the Docstring, for example
'The normalized data is usually used for inspecting the data, e.g., using plots or when extracting information such as the amplitude of harmonic components. Most processing operations, e.g., frequency domain convolution require the non-normalized data.'

The opposite could be added to the `freq_raw` docstring

Reply:
Yes, good point. Changed the docstrings accordingly :)